distribution/ddf-common/src/main/resources/common-bin.xml

```diff
@@ -51,6 +51,26 @@
                 <exclude>RELEASE-NOTES</exclude>
                 <exclude>karaf-manual*.html</exclude>
                 <exclude>karaf-manual*.pdf</exclude>
+
+                <!-- excluding jars we don't need that have OWASP vulnerabilities -->
+                <!--
+                Excluding this unused jar, as we don't use it (TO BE CONFIRMED)
+                and it's associated with CVE-2004-0009
+                -->
+                <exclude>**/bundles/org.apache.servicemix.bundles.not-yet-commons-ssl/**</exclude>
```

Comment:
Now that I look at the CVE it looks unrelated to this bundle OWASP is picking up anything with apache and ssl in the name. Ill remove this and add a suppression.